repos/fdbt-site/src/pages/ticketConfirmation.tsx

```diff
@@ -401,30 +434,37 @@ export const buildTicketConfirmationElements = (
     ctx: NextPageContextWithSession,
 ): ConfirmationElement[] => {
     let confirmationElements: ConfirmationElement[];
-
-    switch (fareType) {
-        case 'single':
-            confirmationElements = buildSingleTicketConfirmationElements(ctx);
-            break;
-        case 'return':
-            confirmationElements = buildReturnTicketConfirmationElements(ctx);
-            break;
-        case 'period':
-            confirmationElements = buildPeriodOrMultiOpTicketConfirmationElements(ctx);
-            break;
-        case 'flatFare':
-            confirmationElements = buildFlatFareTicketConfirmationElements(ctx);
-            break;
-        case 'multiOperator':
+    if (fareType === 'period') {
```

Comment:
Can you include this in the switch statement please? Reactor it into a function I think to make it not a special case